h2o-r/tests/testdir_algos/word2vec/runit_text8_test_large.R

```diff
@@ -0,0 +1,21 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+
+
+
+test.word2vec.sg_hs <- function() {
+  text8.path = locate("bigdata/laptop/text8.gz")
+
+  words <- h2o.importFile("/Users/mkurka/git/h2o/h2o-3/bigdata/laptop/text8.gz",
+  destination_frame = "words", col.names = "word", col.types = "String")
+  w2v <- h2o::h2o.word2vec(words, minWordFreq = 5, vecSize = 50, sentSampleRate = 0.001,
```

Comment:
why reason why using `h2o::` ? h2o should be already attached. Unless there is other package that mask this function by own with the same name there is no benefit from using it here. Also using it is disabling ability for developers to spawn new version of `h2o.word2vec` and run unit test on it.